File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    for i in range(0, len(array_to_search_through)):
        if array_to_search_through[i] == value_to_find:
            return array_to_search_through[i]
    return None
print(linear_search(4, [1, 3, 4]))

File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {

    for (let i = 0; i < arrayToSearchThrough.length; i++){
        if (arrayToSearchThrough[i] == valueToFind){
            return arrayToSearchThrough[i];
        }
    }
    return undefined
};

function GlobalSearch(value, oldArray){
    let newArray = [];
   
    for (let i = 0; i < oldArray.length; i++){
        if (oldArray[i] == value){
            newArray.push(i);
        }
        
    }
    return newArray;


}
// // console.log(this.linearSearch(3, [1, 2, 3]));
// console.log(GlobalSearch("a", ["b", "a", "n", "a", "n", "a", "s"]))


